CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+## Before You Start Coding
+
+- Set up `git`, `node.js` and this project as described in [the README](./README.md)
+- Create a new branch with the name `<your-name>/<feature>` with `git checkout -b <your-name>/<feature>`
+- Style note: Branch names should be short. For instance something like `vivianwilde/contributing-guide` would be a good branch name
+- Make your changes to the code, browse stackoverflow in despair, etc. etc. (basically, the usual development steps)
```

Comment:
I don't think this line is necessary. It's a bit of a given and we're also instructing the reader to code in the `before you start coding`  section.